src/components/navbar/Navbar.tsx

```diff
@@ -1,33 +1,51 @@
+import React, { useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
 import NavLink from "components/UI/navlink/NavLink";
 import NavbarStyles, { NavBarButtons } from "components/navbar/NavbarStyles";
 import logo from "components/navbar/imgs/logo.svg";
-import useViewport from "hooks/useViewport";
-import NavBarButton from "components/UI/buttons/navBarButton/NavBarButton";
-import MenuDrawer from "components/UI/drawers/navBarDrawer/MenuDrawer";
-import { MessageFilled, LogoutOutlined } from "@ant-design/icons";
+import { BellOutlined, LogoutOutlined, UserOutlined } from "@ant-design/icons";
 import {
-  CHAT_ROUTE,
   SIGN_UP_ROUTE,
   SIGN_IN_ROUTE,
+  CLIENT_PROFILE,
 } from "utils/consts/routeConsts";
 import useAuth from "hooks/useAuth";
-import { MOBILE_SCREEN_SIZE } from "utils/consts/navBarConsts";
-import { useEffect, useState } from "react";
-import navLinksPerRole, {
-  NavLinkOptions,
-} from "components/navbar/NavLinksPerRole";
+import { Badge, Menu } from "antd";
+import { Link } from "react-router-dom";
+import {
+  useGetNotificationsQuery,
+  getSocket,
+} from "services/notifications/setNotificationsAPI";
+import NavBarButton from "components/UI/buttons/navBarButton/NavBarButton";
+import navLinksPerRole, { NavLinkOptions } from "./NavLinksPerRole";
 
 function Navbar() {
   const { authToken, signOut, role } = useAuth();
-  const [navItens, setNavItens] = useState<NavLinkOptions | null>();
+  const { data, isError, isLoading } = useGetNotificationsQuery();
   const { t } = useTranslation();
-  const { screenWidth } = useViewport();
+
+  const [navItens, setNavItens] = useState<NavLinkOptions | null>();
+  const [countNotifications, setCountNotifications] = useState(0);
```

Comment:
Can u define generic `useState<number>`. I understand thats it obvious, just to keep in same code style, like in line 27